tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_Constraints.cpp

```diff
@@ -798,22 +801,25 @@ void test_constraint_compute_items(
   const auto& deriv_pi =
       get<Tags::deriv<GeneralizedHarmonic::Tags::Pi<3, Frame::Inertial>,
                       tmpl::size_t<3>, Frame::Inertial>>(gh_derivs);
+  get<Tags::deriv<GeneralizedHarmonic::Tags::Pi<3, Frame::Inertial>,
```

Comment:
what is this doing?

Reply:
Nothing. Good catch! Looks like a copy/paste typo. I'll remove it

Reply:
I'll just squash this change because it's a one-line fix